pkg/cosign/kubernetes/apis/v1alpha1/clusterimagepolicy_types.go

```diff
@@ -0,0 +1,70 @@
+// +kubebuilder:validation:Optional
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/apis"
+)
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:path=clusterimagepolicies,scope=Cluster
+// +kubebuilder:storageversion
+type ClusterImagePolicy struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              ClusterImagePolicySpec `json:"spec,omitempty"`
+}
+
+var (
+	_ apis.Validatable = (*ClusterImagePolicy)(nil)
+	_ apis.Defaultable = (*ClusterImagePolicy)(nil)
+)
+
+type ClusterImagePolicySpec struct {
+	Images []ImagePattern `json:"images,omitempty"`
```

Comment:
I don't think the Images field should be omitted if empty.

Reply:
We've dropped `omitempty` for now.